hypervisor/arch/x86/guest/vm.c

```diff
@@ -930,8 +935,14 @@ void prepare_vm(uint16_t vm_id, struct acrn_vm_config *vm_config)
 #ifdef CONFIG_SECURITY_VM_FIXUP
 	security_vm_fixup(vm_id);
 #endif
+	if (get_vmid_by_name(vm_config->name) == ACRN_INVALID_VMID) {
```

Comment:
IN case, we have 2 vms with same name, we can identify thru   get_vmid_by_name != vm_id.

Reply:
for pre-launched VM, check  get_vmid_by_name != vm_id is ok,
for post-launched VM, the vmid is unknow before call  get_vmid_by_name

Reply:
for post_launched VM, you are doing check inside parse_target_vm() --- if the returned vm_id is an created VM (you can check the VM state), that is name duplication. If it is a new VM_id, It already eliminated the duplicated name issues.

Reply:
will update the patch